database/src/pool/sqlite.rs

```diff
@@ -385,7 +385,23 @@ static MIGRATIONS: &[Migration] = &[
         alter table pstat_series_new rename to pstat_series;
     "#,
     ),
-    Migration::new("alter table pull_request_build add column backends text"),
+    // Add target as a unique constraint, defaulting to 'x86_64-unknown-linux-gnu'
+    Migration::without_foreign_key_constraints(
+    r#"
+        create table pstat_series_with_target(
+            id integer primary key not null,
+            crate text not null references benchmark(name) on delete cascade on update cascade,
+            profile text not null,
+            scenario text not null,
+            backend text not null,
+            metric text not null,
+            target text not null default 'x86_64-unknown-linux-gnu',
```

Comment:
Since we're recreating the table, might as well again keep the metric column at the end here, it kind of makes more sense (id -> name of the benchmark -> parameters of the benchmark -> result of the benchmark).